src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1715,6 +1724,81 @@ private void SetPeriodicEvaluation(WorkspaceModel ws)
             return true;
         }
 
+        // Reload's the dummy nodes and serializes the new node information once the package resolves it. 
+        private void ReloadDummyNodes()
+        {
+            JObject dummyNodeJSON = null;
+            WorkspaceModel currentWorkspace = this.CurrentWorkspace;
+
+            if (currentWorkspace == null || string.IsNullOrEmpty(currentWorkspace.FileName))
```

Comment:
what happens if this is a customNode workspace? Have you tested that? I think we need a test for that case.